main/models.py

```diff
@@ -0,0 +1,66 @@
+from django.db import models
+
+
+class User(models.Model):
```

Comment:
Please use [Django user model](https://docs.djangoproject.com/en/4.0/ref/contrib/auth/#user-model) or create a custom user model as described [here](https://docs.djangoproject.com/en/4.0/topics/auth/customizing/#specifying-a-custom-user-model). Please add `timestamp`, `date_of_birth`, `email` fields.

Reply:
I made class UserProfile, field user is OneToOneField to Django user model, field 'email' exists in that model, date_of_birth and timestamp have been added to UserProfile